gitclone/checkout_method_pr_auto_merge_branch.go

```diff
@@ -0,0 +1,66 @@
+package gitclone
+
+import (
+	"fmt"
+
+	"github.com/bitrise-io/go-utils/command/git"
+	"github.com/bitrise-io/go-utils/log"
+)
+
+//
+// checkoutPRMergeBranch
+type checkoutPRMergeBranch struct {
+	// baseBranch string
+	// // Merge branch contains the changes already merged
+	// mergeBranch string
+	params PRMergeBranchParams
```

Comment:
Please move `PRMergeBranchParams` in this file

Reply:
done